src/main/java/mil/dds/anet/beans/Report.java

```diff
@@ -32,6 +33,10 @@
     APPROVED
   }
 
+  public enum EngagementStatus {
+    HAPPENED, FUTURE, CANCELLED
+  }
```

Comment:
Please import this from `ReportSearchQuery` (or import it from here in that file). We now get:
> WARN io.leangen.graphql.generator.OperationMapper: Potential type name collision detected: 'EngagementStatus' bound to multiple types: mil.dds.anet.beans.search.ReportSearchQuery$EngagementStatus (loaded by sun.misc.Launcher$AppClassLoader@4e0e2f2a) and mil.dds.anet.beans.Report$EngagementStatus (loaded by sun.misc.Launcher$AppClassLoader@4e0e2f2a). Assign unique names using the appropriate annotations or override the TypeInfoGenerator.